draft-packages/badge/docs/Badge.stories.tsx

```diff
@@ -1,5 +1,5 @@
-import React from "react"
-import { Button } from "@kaizen/draft-button"
+import * as React from "react"
```

Comment:
```suggestion
import React from "react"
```

Reply:
ah drat, I think that got override in a conflict change, will fix that up 👍